tests/integration_tests/test_csv_splitter_integration.py

```diff
@@ -0,0 +1,23 @@
+from main import create_files
+from tests.mock_data import CHUNKS
+import pytest
+import os
+
+
+@pytest.fixture
+def create_files_from_chunks():
```

Comment:
I don't want you to use "actual" code (`create_files`) in tests fixture functions.
If there are setup or teardown functions needed for tests, we define them separately in some designated module and treat them as code that can be relied on.
My problem was not that the **test** is in the same file as the other tests, but rather that you're using a fixture to test your code.